index.d.ts

```diff
@@ -0,0 +1,19 @@
+declare module "react-broadcast" {
+  import { ComponentClass } from "react"
+
+  namespace ReactBroadcast {
+    interface BroadcastProps {
+      channel: string,
+      value?: any
+    }
+
+    interface SubscriberProps {
+      channel: string
```

Comment:
Could you make these generic on the value type and specify the `children` prop as `(value: T) => React.ReactNode`?